chain/vm/gas_v0.go

```diff
@@ -120,6 +120,8 @@ type pricelistV0 struct {
 	verifyPostLookup     map[abi.RegisteredPoStProof]scalingCost
 	verifyPostDiscount   bool
 	verifyConsensusFault int64
+
+	verifuReplicaUpdate int64
```

Comment:
```suggestion
	verifyReplicaUpdate int64
```

Reply:
That is what I get for fixing typos after I build and test.